CleverTapSDK/InApps/CTInAppEvaluationManager.m

```diff
@@ -298,7 +330,7 @@ - (void)updateTTL:(NSMutableDictionary *)inApp {
 - (BatchHeaderKeyPathValues)onBatchHeaderCreationForQueue:(CTQueueType)queueType {
     // Evaluation is done for events only at the moment,
     // send the evaluated and suppressed ids in that queue header
-    if (queueType != CTQueueTypeEvents) {
+    if (queueType != CTQueueTypeEvents && queueType != CTQueueTypeProfile) {
```

Comment:
Create separate collections for the profile server-side in-app ids and the profile suppressed client-side in-apps.

Add to the header the ones based on the queue type.